runtime/tests/checker/conformance_test.go

```diff
@@ -45,3 +46,200 @@ func TestCheckInvalidEventTypeRequirementConformance(t *testing.T) {
 
 	require.IsType(t, &sema.ConformanceError{}, errs[0])
 }
+
+func TestCheckTypeRequirementConformance(t *testing.T) {
+
+	type test struct {
+		name            string
+		preparationCode string
+		interfaceCode   string
+		conformanceCode string
+		valid           bool
+	}
+
+	tests := []test{
+		{
+			"Both empty",
+			``,
+			`pub struct S {}`,
+			`pub struct S {}`,
+			true,
+		},
+		{
+			"Conformance with additional function",
+			``,
+			`
+              pub struct S {}
+            `,
+			`
+              pub struct S {
+                  fun foo() {}
+              }
+            `,
+			true,
+		},
+		{
+			"Conformance with missing function",
+			``,
+			`
+              pub struct S {}
+            `,
+			`
+              pub struct S {
+                  fun foo() {}
+              }
+            `,
+			true,
+		},
+		{
+			"Conformance with missing function",
```

Comment:
Duplicate test case

Reply:
Fixed in 6b0124b6d2af5448b9909cbe21a38e0130c7816d